src/components/forms/pool_actions/InvestForm/components/InvestPreviewModal/components/InvestActions.vue

```diff
@@ -78,16 +60,14 @@ const investmentState = reactive<InvestmentState>({
 const route = useRoute();
 const { t } = useI18n();
 const { networkConfig } = useConfig();
-const { getToken } = useTokens();
 const { account, getProvider, explorerLinks } = useWeb3();
 const { addTransaction } = useTransactions();
 const { txListener, getTxConfirmedAt } = useEthers();
 const { parseError } = useTranasactionErrors();
 const { fullAmounts, bptOut, fiatTotalLabel } = toRefs(props.math);
-
-const { requiredApprovalState, approveToken } = useTokenApprovals(
+const { tokenApprovalActions } = useTokenApprovalActions(
   props.tokenAddresses,
-  fullAmounts
+  ref(fullAmounts.value)
```

Comment:
I'm not sure why this has been changed. This is essentially converting a reactive property to a reactive property.

Reply:
lol yea don't know what I was thinking there.